tests/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+set(TESTING_TIMEOUT 2000)
```

Comment:
let's use a time like 1200, which is 20 minutes, and long enough for the python regression test on the slowest VM I've used